src/pages/Search/SearchBooks.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import '../../styles/theme';
+import styled from 'styled-components';
+
+const filters = [
+  { filter: '전체', placeholder: '검색어를 입력하세요' },
+  { filter: '저자', placeholder: '저자를 입력하세요' },
+  { filter: '제목', placeholder: '제목을 입력하세요' },
+  { filter: '출판사', placeholder: '출판사를 입력하세요' },
+];
+
+export default function SearchBooks() {
+  const [filter, setFilter] = useState('전체');
```

Comment:
생각해보면 굳이 filter와 placeholder를 나누어 저장할 필요가 없을 것 같습니다. state 둘 중 하나만 유지하는게 어떨까요?
```suggestion
  const [filter, setFilter] = useState({ type: '전체', placeholder: '검색어를 입력하세요' });
```

Reply:
수정 완료했습니다